consensus/reactor_test.go

```diff
@@ -668,6 +668,7 @@ func capture() {
 // Ensure basic validation of structs is functioning
 
 func TestNewRoundStepMessageValidateBasic(t *testing.T) {
+	t.Parallel()
```

Comment:
what is the benefit of adding these instructions?

my opinion is that parallelism should be controlled via test flags (`-test.count or -test.cpu`) and maybe very rarely in a test code.

Reply:
`t.Parallel()` signals that this test can be run in parallel with other tests which are also marked with `t.Parallel()`. But again, this is not the right place to use this, we should call this inside the sub-tests, so the sub-tests can be run parallel with each others.

This is different with using `-test-count` or `-test-cpu`. Those flags cause a single test to be run multiple times, but multiple instances of a single test never run in parallel with each other.